.github/workflows/python-package.yml

```diff
@@ -27,15 +27,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8 pytest
         python -m pip install .
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        python -m pip install -r requirements.dev.txt
```

Comment:
I think that you should keep the requirements.txt, else the current package won't find the needed dependencies to run. Why do you think it's not useful ?

Reply:
the requirements are already specified in the setup.py which is installed as a part of `python -m pip install .`

Reply:
We could also remove the `requirements.dev.txt` and have them as an extra similar to [transformers](https://github.com/huggingface/transformers/blob/00c1d87a7d5c8dfb4554370983b5a3f7c069edd7/setup.py#L445C20-L445C26) ; So one would install dev with something like `pip install -e ".[dev]"` (https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md); I don't mind either one, the current way may actually use less code & is simpler so I think it's fine

Reply:
I will change it to the ".[dev]" approach. That was all requirements are specified in one place.